features/tracks/src/main/kotlin/com/wachon/spotiwrap/features/tracks/presentation/common/TrackItem.kt

```diff
@@ -37,12 +54,51 @@ fun TrackItem(
                 .size(100.dp),
         )
         Spacer(modifier = Modifier.height(8.dp))
-        TextNoPadding(
-            text = track.trackTitle,
-            style = Body.copy(fontWeight = FontWeight.W600),
-            maxLines = 1,
-            overflow = TextOverflow.Ellipsis
-        )
+        Box(
+            modifier = Modifier
+                .fillMaxWidth()
+                .height(20.dp)
+        ) {
+            TextNoPadding(
+                text = track.trackTitle,
+                style = Body.copy(fontWeight = FontWeight.W600),
+                maxLines = 1,
+                softWrap = false,
+                overflow = TextOverflow.Clip
+            )
+            Box(
+                modifier = Modifier
+                    .height(25.dp)
+                    .fillMaxWidth()
+                    .background(
+                        Brush.horizontalGradient(
+                            colors = listOf(Color.Transparent, Color.Black), startX = 175f, endX = 240f
+                        )
+                    )
+            ) {
+                if (track.trackFame != NONE) {
```

Comment:
This code seems to be repeated twice and maybe its going to be used more. Maybe we could think in making an standalone composable

Reply:
Component created ;)